kong/plugins/response-ratelimiting/schema.lua

```diff
@@ -75,6 +75,9 @@ return {
           { redis_port = typedefs.port({ default = 6379 }), },
           { redis_password = { type = "string", len_min = 0, referenceable = true }, },
           { redis_username = { type = "string", referenceable = true }, },
+          { redis_ssl = { type = "boolean", required = true, default = false, }, },
+          { redis_ssl_verify = { type = "boolean", required = true, default = false }, },
```

Comment:
i'm sorry about it, so,

this is a required field should be a breaking change? 🤔 
I mean that we should set as required false, and check on the implementation 🤔 

thank you guy for the commit!

Reply:
You are absolutely right! For it to be backward compatible it has to be required=false. I pushed a commit!

Reply:
Is it ok like that, or do I need to change something else?

Reply:
Hey, any chance this will go in the next release? :)

Reply:
I don't think this is a breaking change.
Although `redis_ssl` and `redis_ssl_verify` are set as required fields, they have default values, so users can omit them.

See the following two codes.
https://github.com/Kong/kong/blob/b5bd80d64201c2908ff509ad30b3f9ce08730b1d/kong/plugins/rate-limiting/schema.lua#L88-L89 https://github.com/Kong/kong/blob/b5bd80d64201c2908ff509ad30b3f9ce08730b1d/spec/03-plugins/23-rate-limiting/05-integration_spec.lua#L140-L147

These two fields are required, but they have default values, so these two fields can be omitted.

Reply:
👍 I made them mandatory again